dt-core/admin/menu/tabs/tab-workflows-docs.php

```diff
@@ -0,0 +1,12 @@
+<!-- Here you can create different Help Texts for use in different places -->
+
+<!-- Comments Field -->
+<div id="dt_right_docs_comments_title" style="display: none;">
+    Comments Field
+</div>
+<div id="dt_right_docs_comments_content" style="display: none;">
+    When creating comments, you can insert field values from the current record. This can be helpful for showing what has changed or for tagging users who are selected in a given field. The following fields are supported for replacement using the given replacement tokens.
+    <ul id="dt_right_docs_comments_list" style="height: 300px; overflow: auto;">
```

Comment:
This overflow was a good idea. See my comment on the CSS in the main workflow file for how we can move it to scroll the whole side column. So we can remove these inline styles.